app/src/main/java/io/gnosis/safe/ui/transactions/details/models/TransactionDetailsViewData.kt

```diff
@@ -0,0 +1,162 @@
+package io.gnosis.safe.ui.transactions.details.models
```

Comment:
I would rename models package to viewdata

Reply:
Done in d8a57974a043ec9d82ebcce0fb951574dcc5eecd